### PR TITLE
New nav item for events

### PR DESCRIPTION
**Pull request by helenb**

### CODE REVIEW

File: tbx/project_styleguide/templates/patterns/molecules/primary-nav/primary-nav.html [hunk 1]
```diff
@@ -26,5 +26,10 @@
                 </a>
             {% endif %}
         </li>
+        <li class="nav-item">
+            <a class="nav-item__link {% if page.slug == 'events' %}nav-item__link--active{% endif %}" href="/events/">
+                <span class="nav-item__title">Events</span>
```
Comment (zerolab) [inline tbx/project_styleguide/templates/patterns/molecules/primary-nav/primary-nav.html:31]: question: why is `span.nav-item__title` necessary on the nav items?

as far as I can tell, moving the `.nav-item__title` styles to `.nav-item__link` (i.e. the `<a>` element) accomplishes the same thing with one less layer of nesting.

ditto for `.mobile-nav-item__title`

Comment (helenb) [reply tbx/project_styleguide/templates/patterns/molecules/primary-nav/primary-nav.html:31]: @zerolab you are almost certainly right... but limited time to refactor today. I've fixed a more pressing bug to remove a nasty horizontal scroll at tablet.

Comment (chris-lawton) [reply tbx/project_styleguide/templates/patterns/molecules/primary-nav/primary-nav.html:31]: I've refactored this in https://github.com/torchbox/wagtail-torchbox/pull/242. Thanks for pointing out Dan 👍

Comment (zerolab) [reply tbx/project_styleguide/templates/patterns/molecules/primary-nav/primary-nav.html:31]: 🌟

**Comment by chris-lawton** [comment]
This looks good to me @helenb. Some of it will be removed as part of the work done in https://github.com/torchbox/wagtail-torchbox/pull/242 (hooking up the menu to wagtail) however it's still worth getting this in for the styling and breakpoint updates.

Worth nothing what Dex has put on the ticket:

> The menu item is the last thing that needs to go live for Events to show. Putting this off atm as we have no events planned currently

Might be worth waiting for https://github.com/torchbox/wagtail-torchbox/pull/242 to be merged thinking about it, as then we can add the link in for the events page via wagtail when ready.

**Comment by chris-lawton** [comment]
@helenb i've ported the styles you've added here to #242 where the FE and BE for the navigation is being hooked up (https://github.com/torchbox/wagtail-torchbox/pull/242/commits/adf3ce7e21ef3e1aee17a68c5cf6d9e18d52f3a8) so to avoid conflicts.

**Pull request closed without merging**